splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/SingleInputComponent.jsx

```diff
@@ -22,15 +22,17 @@ function SingleInputComponent(props) {
         allowList,
         placeholder = _('Select a value'),
         dependencies,
+        // eslint-disable-next-line no-unused-vars
         createSearchChoice,
         referenceName,
+        // eslint-disable-next-line no-unused-vars
         disableSearch,
         labelField,
         autoCompleteFields,
     } = controlOptions;
 
-    function handleChange(e, { value }) {
-        restProps.handleChange(field, value);
+    function handleChange(e, { val }) {
```

Comment:
@dkhatri-crest Did you validated this? I think it won't work.

Reply:
updated